dropwizard-migrations/src/test/java/io/dropwizard/migrations/CloseableLiquibaseTest.java

```diff
@@ -25,7 +29,9 @@ public void setUp() throws Exception {
         factory.setUser("DbTest");
 
         dataSource = (ManagedPooledDataSource) factory.build(new MetricRegistry(), "DbTest");
-        liquibase = new CloseableLiquibaseWithClassPathMigrationsFile(dataSource, "migrations.xml");
+        DatabaseConnection conn = new JdbcConnection(dataSource.getConnection());
+        Database database = DatabaseFactory.getInstance().findCorrectDatabaseImplementation(conn);
+        liquibase = new CloseableLiquibaseWithClassPathMigrationsFile(dataSource, database, "migrations.xml");
```

Comment:
Made this change, and fixed the test a bit to actually check for an active conn. before closing (it wasn't an effective test before).